storage_service/locations/fixtures/package.json

```diff
@@ -44,7 +44,7 @@
         "size": 0,
         "package_type": "Transfer",
         "status": "Uploaded",
-        "misc_attributes": ""
+        "misc_attributes": "{}"
```

Comment:
Empty string is not a valid JSON document, causing the fixture loader to break. This probably started happening when we updated `jsonfield`.